docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - DEBUG=1
       - DATA_DIR=/tmp/localstack/data
       - DOCKER_HOST=unix:///var/run/docker.sock
-      - LAMBDA_EXECUTOR=docker
+      - LAMBDA_EXECUTOR=docker-reuse
```

Comment:
This reuses a single docker container to prevent cold start time on every lambda invocation. 🚀